### PR TITLE
DTSSTCI-984: Added integration tests for Caseworker manage case flag and maintain link case

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### Change description ###

Added

### JIRA link ###

https://tools.hmcts.net/jira/browse/DTSSTCI-984

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)
- [ ] `enable-e2e-tests` label can be used to run the e2e tests before QA handover and before release (required)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket has been reviewed by QA
- [ ] the user story has been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerManageCaseFlagIT.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        IdamWireMock.start();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        IdamWireMock.stopAndReset();
+    }
+
+    @Test
+    void shouldReturnSuccessMessageOnAboutToSubmit() throws Exception {
```
Comment (tomxelliott) [inline src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerManageCaseFlagIT.java:71]: Do we want to specify what it is that happens on about to submit in the title?

**Review by tomxelliott — DISMISSED** [review]
Looks good, just a small comment on the about to submit test name

File: src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerManageCaseFlagIT.java (hunk 1 above)
Comment (rajaram-nagarajan) [reply src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerManageCaseFlagIT.java:71]: tbh, I was wondering whether this abouttosubmit call back is needed as I don't see it doing anything :D

Comment (rajaram-nagarajan) [reply src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerManageCaseFlagIT.java]: Removed the dummy abouttosubmit callback

**Review by gmmagruder — APPROVED** [review]

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**